### PR TITLE
Refactor part 3 - generate more actions - login

### PR DESCRIPTION
**Pull request by chrisnojima**

### CODE REVIEW

File: shared/reducers/login.js [hunk 1]
```diff
@@ -97,58 +97,58 @@ export default function(state: Constants.State = initialState, action: any): Con
...
+    case LoginGen.configuredAccounts:
       if (action.payload.error) {
         return {...state, configuredAccounts: []}
       } else {
         return {...state, configuredAccounts: action.payload.accounts}
       }
-    case Constants.waitingForResponse:
+    case LoginGen.waitingForResponse:
       return {...state, waitingForResponse: action.payload}
-    case Constants.loginDone:
```
Comment (chrisnojima) [inline shared/reducers/login.js]: loginDone actually had the same constant as relogin and it never sets the error

File: shared/actions/login/saga.js [hunk 2]
```diff
@@ -474,7 +473,6 @@ function* loginFlowSaga(usernameOrEmail, passphrase) {
         console.log(error)
         yield Saga.call(handleProvisioningError, error)
       } else {
-        yield Saga.put(Creators.loginDone())
```
Comment (chrisnojima) [inline shared/actions/login/saga.js]: i *think* this does nothing

**Comment by chrisnojima** [comment]
@keybase/react-hackers this is ready. please test out this code since it touches the login flow a bit

File: shared/actions/login/saga.js (hunk 2 above)
Comment (MarcoPolo) [reply shared/actions/login/saga.js]: :+1: Delete the generated code too

File: shared/login/register/code-page/container.js [hunk 3]
```diff
@@ -63,12 +64,12 @@ const mapStateToProps = ({
...
-  onBack: () => dispatch(Creators.onBack()),
-  setCodePageMode: requestedMode => dispatch(Creators.setCodePageMode(requestedMode)),
-  setCameraBrokenMode: (broken: boolean) => dispatch(Creators.setCameraBrokenMode(broken)),
-  qrScanned: ({data}) => dispatch(Creators.qrScanned(data)),
-  resetQRCodeScanned: () => dispatch(Creators.resetQRCodeScanned()),
-  textEntered: text => dispatch(Creators.provisionTextCodeEntered(text)),
+  onBack: () => dispatch(LoginGen.createOnBack()),
+  setCodePageMode: mode => dispatch(LoginGen.createSetCodePageMode({mode})),
+  setCameraBrokenMode: (broken: boolean) => dispatch(LoginGen.createSetCameraBrokenMode({broken})),
+  qrScanned: ({data}) => dispatch(LoginGen.createQrScanned({phrase: new HiddenString(data)})),
```
Comment (MarcoPolo) [inline shared/login/register/code-page/container.js]: Did flow catch this or you remembered?

Comment (chrisnojima) [reply shared/login/register/code-page/container.js]: mmmm don't remember

**Pull request merged**